docs/docs_settings.py

```diff
@@ -7,26 +7,17 @@
 
 import os
 
-if os.environ['EDX_PLATFORM_SETTINGS'] == 'devstack_docker':
-    from lms.envs.devstack_docker import *  # lint-amnesty, pylint: disable=wildcard-import
-    from cms.envs.devstack_docker import (  # lint-amnesty, pylint: disable=unused-import
-        ADVANCED_PROBLEM_TYPES,
-        COURSE_IMPORT_EXPORT_STORAGE,
-        LIBRARY_AUTHORING_MICROFRONTEND_URL,
-        SCRAPE_YOUTUBE_THUMBNAILS_JOB_QUEUE,
-        VIDEO_TRANSCRIPT_MIGRATIONS_JOB_QUEUE,
-        UPDATE_SEARCH_INDEX_JOB_QUEUE,
-    )
-else:
-    from lms.envs.devstack import *  # lint-amnesty, pylint: disable=wildcard-import
-    from cms.envs.devstack import (
-        ADVANCED_PROBLEM_TYPES,
-        COURSE_IMPORT_EXPORT_STORAGE,
-        LIBRARY_AUTHORING_MICROFRONTEND_URL,
-        SCRAPE_YOUTUBE_THUMBNAILS_JOB_QUEUE,
-        VIDEO_TRANSCRIPT_MIGRATIONS_JOB_QUEUE,
-        UPDATE_SEARCH_INDEX_JOB_QUEUE,
-    )
+from lms.envs.devstack import *  # lint-amnesty, pylint: disable=wildcard-import
+from cms.envs.devstack import (  # lint-amnesty, pylint: disable=unused-import
+    ADVANCED_PROBLEM_TYPES,
+    COURSE_IMPORT_EXPORT_STORAGE,
+    GIT_EXPORT_DEFAULT_IDENT,
```

Comment:
why added?

Reply:
Good q; `make guides` was failing without it, even on master:

```
Configuration error:
There is a programmable error in your configuration file:

Traceback (most recent call last):
  File "/edx/app/edxapp/venvs/edxapp/lib/python3.8/site-packages/sphinx/config.py", line 319, in eval_config_file
    execfile_(filename, namespace)
  File "/edx/app/edxapp/venvs/edxapp/lib/python3.8/site-packages/sphinx/util/pycompat.py", line 89, in execfile_
    exec(code, _globals)
  File "/edx/app/edxapp/edx-platform/docs/guides/conf.py", line 41, in <module>
    django.setup()
  File "/edx/app/edxapp/venvs/edxapp/lib/python3.8/site-packages/django/__init__.py", line 24, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/edx/app/edxapp/venvs/edxapp/lib/python3.8/site-packages/django/apps/registry.py", line 122, in populate
    app_config.ready()
  File "/edx/app/edxapp/edx-platform/cms/djangoapps/contentstore/apps.py", line 23, in ready
    from .signals import handlers  # pylint: disable=unused-import
  File "/edx/app/edxapp/edx-platform/cms/djangoapps/contentstore/signals/handlers.py", line 17, in <module>
    from cms.djangoapps.contentstore.proctoring import register_special_exams
  File "/edx/app/edxapp/edx-platform/cms/djangoapps/contentstore/proctoring.py", line 23, in <module>
    from .views.helpers import is_item_in_course_tree
  File "/edx/app/edxapp/edx-platform/cms/djangoapps/contentstore/views/__init__.py", line 9, in <module>
    from .export_git import *
  File "/edx/app/edxapp/edx-platform/cms/djangoapps/contentstore/views/export_git.py", line 15, in <module>
    import cms.djangoapps.contentstore.git_export_utils as git_export_utils
  File "/edx/app/edxapp/edx-platform/cms/djangoapps/contentstore/git_export_utils.py", line 24, in <module>
    GIT_EXPORT_DEFAULT_IDENT = settings.GIT_EXPORT_DEFAULT_IDENT
  File "/edx/app/edxapp/venvs/edxapp/lib/python3.8/site-packages/django/conf/__init__.py", line 80, in __getattr__
    val = getattr(self._wrapped, name)
AttributeError: 'Settings' object has no attribute 'GIT_EXPORT_DEFAULT_IDENT'
```

I don't think we build these docs in CI or publish them anywhere, so this probably just organically broke at some point in the past :/